### ISSUE TITLE
WXM Earnings Graph does not show earnings on console

### ISSUE THREAD

**Issue by DakotaIRL** [WeatherXM/issue-tracker#87]
## WeatherXM Related Element
- Website

## **Describe the bug**
The earnings graph does not update when new earnings are granted. 
New earnings are being added to the 'WXM Earnings' section but the daily breakdown does not update accordingly.
Only the first day of deployment earnings are listed.

### **How To Reproduce**
'Cheery Fuzzy Cirrostratus'
1. Go to 'console.weatherxm.com'
2. View Earnings Graph
4. See Error

### **Expected behavior**
I expect the graph to update daily when new earnings are awarded according to the amount awarded.

### **Screenshots**
![image](https://user-images.githubusercontent.com/108199598/175792344-e828a08e-abb7-4775-b5e6-922bef4abcc8.png)
![image](https://user-images.githubusercontent.com/108199598/175792187-f8153bc2-dba3-471e-9b69-019b13c5f459.png)

### **If Applicable: Desktop:**
 - OS: Windows 10
 - Browser: Brave
 - Version: 1.40.107

### **If Applicable: M5:**
 - Firmware Version 0.3.22

### **Additional context**
You guys are awesome :)

**Comment by RevNotDev** [comment]
Have you changed this to 7 days?
![image](https://user-images.githubusercontent.com/46647866/175793086-7c7269c2-86ff-43f0-b56c-c145298dc860.png)

**Comment by PavlosTze** [comment]
@DakotaIRL Have you tried the above comment of @RevNotDev ? In any case I am passing this to the team.

**Comment by DakotaIRL** [comment]
Yep just tried it and it worked now it shows everything. Thanks.
